src/core/platform.h

```diff
@@ -447,7 +447,7 @@ extern int nni_plat_udp_sockname(nni_plat_udp *, nni_sockaddr *);
 // in APIs to transport file descriptors, or across a fork/exec boundary (so
 // that child processes may use these with socket to inherit a socket that is
 // connected to the parent.)
-extern int nni_socket_pair(int *);
+extern int nni_socket_pair(int [2]);
```

Comment:
It seems a warning of nng.

Reply:
whoops, this shall not be modified

Reply:
nng already fixed this in https://github.com/nanomsg/nng/pull/1760

Reply:
then there shall be sth wrong on sync....